netci.groovy

```diff
@@ -229,7 +229,10 @@ def static getJobName(def configuration, def architecture, def os, def scenario,
             // These are cross builds
             baseName = architecture.toLowerCase() + '_cross_' + configuration.toLowerCase() + '_' + os.toLowerCase()
             break
-        case 'x86':
+        case 'x86ryujit':
```

Comment:
Long term, RyuJIT will inherit the official 'x86' name.  What's the downside to awarding it now?  I can imagine there are reasons; I just can't think of them off hand.